docs/guides/plugins.md

```diff
@@ -251,7 +251,7 @@ class Advanced extends Plugin {
   }
 
   updateState() {
-    this.setState({playing: !player.paused()});
+    this.setState({playing: !this.paused()});
```

Comment:
this probably needs to be `this.player().paused()`

Reply:
according to docs (and my experience) it's `player`:

> With advanced plugins, the value of `this` is the _instance of the plugin class_. The player is passed to the plugin constructor as its first argument (and is automatically applied to the plugin instance as the `player` property) and any further arguments are passed after that.

Reply:
Yup, it's a property, was thinking of regular components.